lib/rmt/import_script.rb

```diff
@@ -0,0 +1,161 @@
+require 'csv'
+require 'ostruct'
+
+
+class ImportScript
+  attr_accessor :config
+
+  class ExitScript < StandardError
+  end
+
+  def initialize(config)
+    @config = config
+  end
+
+  def read_csv(file)
+    # set the quote char to something not used to make sure the csv parser is not interfering with the
+    # JSON quoting.
+    CSV.open(File.join(config.data_dir, file + '.csv'), 'r', { col_sep: "\t", quote_char: "\x00" })
```

Comment:
`quote_char: "\x00"` seems fishy. I'm pretty sure MySQL escapes the data in CSV output, and not with `\x00` symbol.

Reply:
MySQL does not actually. 
```
SCC_xxxxxxxxxxxxxxxx	machinedata	{"hwinfo":{"sockets":1,"arch":"x86_64","cpus":8,"hostname":"e9","hypervisor":"vertical","uuid":null},"distro_target":"sle-12-x86_64","hostname":"e9"}
```
The problem is that the csv parser fails with the quoted strings in the json dump thats why I set the quote_char to something unlikely to happen and yes this is hacky, if you have another suggestions I'm more than happy to change this :)